lib/proxy/plugin_initializer.rb

```diff
@@ -290,6 +301,16 @@ def log_used_settings(settings)
     sorted_keys = settings.keys.map(&:to_s).sort.map(&:to_sym) # ruby 1.8.7 doesn't support sorting of symbols
     to_log = sorted_keys.map {|k| "'%s': %s%s" % [k, settings[k], (default_settings.include?(k) && default_settings[k] == settings[k]) ? " (default)" : ""] }.join(", ")
     logger.debug "'%s' settings: %s" % [plugin.plugin_name, to_log]
+    logger.debug("'%s' ports: 'http': %s, 'https': %s" % [plugin.plugin_name,
```

Comment:
What's the added value of logging this? We already have the enabled option logged in the logs themselves. Or am I missing something here?

Reply:
I had people ask me about this. If anything, this is more of a clarification.